src/bitcoin-cli.cpp

```diff
@@ -452,6 +452,7 @@ class NetinfoRequestHandler : public BaseRequestHandler
         if (conn_type == "block-relay-only") return "block";
```

Comment:
From the PR main thread https://github.com/bitcoin/bitcoin/pull/29415#issuecomment-1986011938:

> The private broadcast succeeds frequently but not always

In the originator's log you can check to whom is the transaction being sent. Then in the recipient's logs you can check if it was accepted and further broadcast to everybody. Enable `-debug=net -debug=privatebroadcast`.